comm/src/main/scala/coop/rchain/comm/comm.scala

```diff
@@ -0,0 +1,34 @@
+package coop.rchain.comm
+
+import scala.util.Try
+
+trait Comm {
+  def send(data: Seq[Byte], p: PeerNode): Try[Boolean]
+  def recv: Try[Seq[Byte]]
+}
+
+case class NodeIdentifier(pKey: Seq[Byte]) {
+  def keccak256(iterations: Int = 1): Seq[Byte] =
+    // Not really hashing, yet.
+    Vector[Byte](pKey: _*)
+
+  def key = keccak256(2)
+}
+
+case class Endpoint(val host: String, val tcpPort: Int, val udpPort: Int) {
```

Comment:
Aren't the parameters to a case class constructor modified with a val by default?  Is there a motivation to make them explicit when they were not made explicit in NodeIdentifier?

Reply:
I think you're right, and these are left over from when I took that encouragement above.